### ISSUE TITLE
Wrong new line check

### ISSUE THREAD

**Issue by kozaxinan** [pinterest/ktlint#842]
The bug you're experiencing might have already be reported!
Please search in the [issues](https://github.com/pinterest/ktlint/issues) before creating one.

## Expected Behavior

Method after multiline text

## Observed Behavior

```  
      """
            |longtext longtext longtext longtext longtext longtext
            |longtext longtext longtext longtext longtext longtext
            |longtext longtext longtext longtext longtext longtext
            |longtext longtext longtext longtext longtext longtext
            """.trimMargin("|")
```

Ktlint gives 2 error.

`Argument should be on a separate line (unless all arguments can fit a single line)ktlintTestSourceSetCheck`
 `Missing newline before ")"`

## Steps to Reproduce

* Version of ktlint used: `0.38.0-alpha01`
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): `ktlintCheck` from https://github.com/JLLeitschuh/ktlint-gradle
* Version of Gradle used (if applicable): 6.6
* Operating System and version: MacOs - 10.15.6
* Link to your project (if it's a public repository):

**Comment by shashachu** [comment]
Thanks for the report @kozaxinan. To clarify: both errors are reporting on the `""".trimMargin("|")` line? And can you also confirm that this is a regression with the 0.38.0 alpha? (i.e. didn't occur in 0.37.2).

**Comment by shashachu** [comment]
Likely introduced by this commit https://github.com/pinterest/ktlint/commit/a94b43fa8978a1cbe882b09b8c6112e612fb66b2

**Comment by kozaxinan** [comment]
Yes that is a recession in alpha and yes it is on trimIndent line.

**Comment by Tapchicoma** [comment]
@kozaxinan with 0.39.0 release you could disable experimental 'argument-list-wrapping' rule to suppress this issue.
